L1Trigger/L1THGCal/interface/be_algorithms/HGCalClusteringImpl.h

```diff
@@ -1,19 +1,24 @@
 #ifndef __L1Trigger_L1THGCal_HGCalClusteringImpl_h__
 #define __L1Trigger_L1THGCal_HGCalClusteringImpl_h__
 
+#include <array> 
+#include <unordered_set>
+#include <unordered_map>
```

Comment:
`unordered_set` and `unordered_map` are not needed in the header file, they are only used internally. Put these in the .cc instead.